### PR TITLE
Simple code and typo fixed in in pkg/scheduler/apis/config/v1alpha1

### PR DESCRIPTION
**Pull request by denkensk**
**What this PR does / why we need it**:
no

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
no

**Special notes for your reviewer**:
no

**Release note**:
no

```release-note
no
```

### CODE REVIEW

File: pkg/scheduler/apis/config/v1alpha1/defaults.go [hunk 1]
```diff
@@ -34,8 +34,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-// SetDefaults_KubeSchedulerConfiguration sets additional defaults
-func SetDefaults_KubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
+// SetDefaultsKubeSchedulerConfiguration sets additional defaults
+func SetDefaultsKubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
```
Comment (sttts) [inline pkg/scheduler/apis/config/v1alpha1/defaults.go]: the underscore style used everywhere else (and maybe even required by the generator)

Comment (denkensk) [reply pkg/scheduler/apis/config/v1alpha1/defaults.go]: sorry, You are right, the underscore style is used everywhere。 I close this。

**Pull request closed without merging**